### PR TITLE
Generalization of two qubit protocols to iSWAP gate

### PR DESCRIPTION
**Pull request by andrea-pasquale**
In this PR, I generalize both `chevron_*` and the former `cz_virtualz_*` to support also the `iSWAP` gate.
Here is an example of chevron
http://login.qrccluster.com:9000/8gUvIzYnQU-zY5U7sLaNFA==
and here is an example of the virtual phase correction
http://login.qrccluster.com:9000/KTJEpaDXR02zVAylQVE_Mg==/

Perhaps documentation could be improved but the code is already working as expected.

Checklist:
- [x] Reviewers confirm new code works as expected.
- [x] Tests are passing.
- [x] Coverage does not decrease.
- [x] Documentation is updated.
- [x] Compatibility with Qibo modules (Please edit this section if the current pull request is not compatible with the following branches).
    - [x] Qibo: `master`
    - [x] Qibolab: `main`
    - [x] Qibolab_platforms_qrc: `main`

### CODE REVIEW

File: src/qibocal/protocols/two_qubit_interaction/virtual_z_phases.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 
 @dataclass
-class CZVirtualZParameters(Parameters):
+class VirtualZPhasesParameters(Parameters):
     """CzVirtualZ runcard inputs."""
```
Comment (Jacfomg) [inline src/qibocal/protocols/two_qubit_interaction/virtual_z_phases.py:27]:
```suggestion
    """VirtualZ runcard inputs."""
```

File: src/qibocal/protocols/two_qubit_interaction/virtual_z_phases.py [hunk 2]
```diff
@@ -43,37 +49,44 @@ class CZVirtualZParameters(Parameters):
 
 
 @dataclass
-class CZVirtualZResults(Results):
+class VirtualZPhasesResults(Results):
     """CzVirtualZ outputs when fitting will be done."""
```
Comment (Jacfomg) [inline src/qibocal/protocols/two_qubit_interaction/virtual_z_phases.py:53]:
```suggestion
    """VirtualZ outputs when fitting will be done."""
```

File: src/qibocal/protocols/two_qubit_interaction/virtual_z_phases_signal.py [hunk 3]
```diff
@@ -10,43 +10,43 @@
...
-from .cz_virtualz import _update, create_sequence
-from .utils import order_pair
+from .virtual_z_phases import _plot as _plot_prob
+from .virtual_z_phases import _update, create_sequence
 
 
 @dataclass
-class CZVirtualZSignalParameters(CZVirtualZParameters):
+class VirtualZPhasesSignalParameters(VirtualZPhasesParameters):
     """CzVirtualZ runcard inputs."""
```
Comment (Jacfomg) [inline src/qibocal/protocols/two_qubit_interaction/virtual_z_phases_signal.py:27]:
```suggestion
    """VirtualZ runcard inputs."""
```

File: src/qibocal/protocols/two_qubit_interaction/virtual_z_phases_signal.py [hunk 4]
```diff
@@ -10,43 +10,43 @@
...
 
 
 @dataclass
-class CZVirtualZSignalResults(CZVirtualZResults):
+class VirtualZPhasesSignalResults(VirtualZPhasesResults):
     """CzVirtualZ outputs when fitting will be done."""
```
Comment (Jacfomg) [inline src/qibocal/protocols/two_qubit_interaction/virtual_z_phases_signal.py:32]:
```suggestion
    """VirtualZ outputs when fitting will be done."""
```

File: src/qibocal/protocols/two_qubit_interaction/virtual_z_phases_signal.py [hunk 5]
```diff
@@ -141,5 +144,7 @@ def _plot(
     return figs, fitting_report
 
 
-cz_virtualz_signal = Routine(_acquisition, _fit, _plot, _update, two_qubit_gates=True)
+correct_virtual_z_phases_signal = Routine(
+    _acquisition, _fit, _plot, _update, two_qubit_gates=True
+)
 """CZ virtual Z correction routine."""
```
Comment (Jacfomg) [inline src/qibocal/protocols/two_qubit_interaction/virtual_z_phases_signal.py:150]:
```suggestion
"""Virtual Z correction routine."""
```

**Review by Edoardo-Pedicillo — APPROVED** [review]

**Comment by andrea-pasquale** [comment]
The code now includes also the two qubit gate sweeps for both iSWAP and CZ.
If I have time I will test on hardware otherwise I will probably merge as it is, since it was already tested (see initial comment).

**Pull request merged**